### PR TITLE
Optimize Preview PR CI without duplicate comments

### PR DESCRIPTION
**Pull request by sy-records**

### CODE REVIEW

**Review by saundefined — APPROVED** [review]
Thanks!

**Comment by sy-records** [comment]
@saundefined Please merge later, I haven't tested it successfully yet

File: .github/workflows/pr-preview.yml [hunk 1]
```diff
@@ -20,9 +20,16 @@ jobs:
...
+          comment-author: 'github-actions[bot]'
+
       - uses: peter-evans/create-or-update-comment@v3
         with:
           issue-number: ${{ github.event.number }}
+          comment-id: ${{ steps.fc.outputs.comment-id }}
           edit-mode: 'replace'
           body: |
-            🚀 Deployed on https://web-php-pr-${{ github.event.number }}.preview.thephp.foundation
+            🚀 Commit $(echo ${GITHUB_SHA::8}) Deployed on https://web-php-pr-${{ github.event.number }}.preview.thephp.foundation
```
Comment (sy-records) [inline .github/workflows/pr-preview.yml:35]:
```suggestion
            🚀 Commit ${{ github.sha }} Deployed on https://web-php-pr-${{ github.event.number }}.preview.thephp.foundation
```

image

**Comment by saundefined** [comment]
@sy-records Looks good, let's try :)

**Pull request merged**